spec/app_spec.lua

```diff
@@ -0,0 +1,15 @@
+local app = require("pl.app")
+
+describe("pl.app.lua", function ()
+
+  local invocation = app.lua()
+
+  it("should pick up the arguments used to run this test", function ()
+    assert.is.truthy(invocation:match("lua.+package.+busted"))
```

Comment:
(nitpick) this test is coupled to how busted is invoked to run this test, hence not deterministic. imo the test should preferably be on its own.

Reply:
I had a question about this too, but it seems to be quite deterministic. I tried different invocations (`luarocks test` vs. `busted`, and it got tried on Linux & Windows. There are of course differences in the actual path this is producing, but at a minimum those components seem to be there any time this test is run from busted — which is has to be because it uses busted tooling.

I'm happy to adjust it but I don't have a better idea.